src/result.rs

```diff
@@ -24,7 +24,7 @@ use task::Poll;
 /// # use tiberius::{Client, AuthMethod};
 /// # use std::env;
 /// use futures::{StreamExt, TryStreamExt};
-/// # async fn foo() -> Result<(), Box<dyn std::error::Error>> {
+/// # async fn _foo() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
Not sure how useful this is, but in `std` it's common to add the `#![allow(unused)]` pragma at the top of the file, and in `async-std` we commonly export this as `#[async_std::main]\n async fn main` inside of a `no_run` block. Neither has large benefits over this approach, but figured it might still be of interest.